src/Symfony/Bridge/PhpUnit/bin/simple-phpunit

```diff
@@ -119,6 +119,8 @@ EOPHP
 
 }
 
+$argv = isset($_SERVER['argv']) ? $_SERVER['argv'] : array();
```

Comment:
onle last thing: before this line, add: `global $argv, $arg;`

Reply:
Good catch. Fixed in 6140de9.